carma/launch/environment.launch.py

```diff
@@ -525,7 +525,7 @@ def generate_launch_description():
                 ],
                 remappings=[
                     ("output/track_list", "cooperative_perception_track_list"),
-                    ("input/detection_list", "filtered_detection_list"),
+                    ("input/detection_list", "filtered_detection_list"), #TODO effectivly bypass
```

Comment:
I'm guessing there is no change required here since this has already been tested to work well - can we update this comment with more description of the suggested future change or remove the comment?

Reply:
yes this is not needed